core-api/src/main/java/com/optimizely/ab/config/ProjectConfig.java

```diff
@@ -58,6 +58,7 @@ public String toString() {
     private final String projectId;
     private final String revision;
     private final String version;
+    private final boolean anonymizeIP;
```

Comment:
just for consistency, can you modify `toString` to include `anonymizeIP` as well?